content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Creating a Simple Chat App in React.js using the chat engine API
+---
+
+This article will teach the React.js way of creating a chat application using the chat engine API. It aims at making it less cumbersome compared to when one opts the Node.js and socket.io way instead.
+
+The learner will understand how the chat engine API works and how to use it to create a precise yet functional direct-messaging application.
+
+
+Approximating to other APIs that deliver user responses to the system, and sends back the system's responses to the user, this API works the same way, with better functionalities.
+
+This tool saves you a lot of implementation time otherwise needed on many lines of code.
+Chat engine API can implement; 
+ - A group chat application.
+ - Direct messages chat application that would work like Facebook messenger .
```

Comment:
An introduction is meant to inform reader what to expect, I feel you should remove these paragraphs from your introduction and add them to the main body where applicable.